src/Resources/Resources/help/New-AzDeployment.md

```diff
@@ -124,26 +198,52 @@ Template parameter values that you enter at the command prompt take precedence o
 
 ## EXAMPLES
 
-### Example 1: Use a custom template and parameter file to create a deployment
+### Example 1: Create a deployment at subscription scope with a custom template and parameter file
 ```
-PS C:\> New-AzDeployment -Location "West US" -TemplateFile "D:\Azure\Templates\EngineeringSite.json" -TemplateParameterFile "D:\Azure\Templates\EngSiteParms.json" -TemplateVersion "2.1"
+PS C:\> New-AzDeployment -ScopeType "Subscription" -DeploymentName "deploy-01" -Location "West US" -TemplateFile "D:\Azure\Templates\ServiceTemplate.json" -TemplateParameterFile "D:\Azure\Templates\ServiceParameters.json"
```

Comment:
@Tiano2017 same comment about adding the corresponding output to all of the examples in this file